keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -1782,6 +1782,15 @@ Alpha:
 - Adequate documentation for the changes
 - Minimum viable test cases mentioned in Test Plan section
 
+Beta:
+
+- Supports concurrency limiting upon long-running requests
```

Comment:
I like the goal, but I also see a lot of value in having this on by default even without long running request handling.

@lavalamp would you hold beta on this?

Reply:
I agree with David - it would be nice to have, but I don't think it should block beta. It would already be huge win without us.

Reply:
I won't block beta on this but I am looking for someone to add this. If possible I'd like it.

Reply:
watch requests (iff served by the cache) is relatively "cheap" in terms of apiserver's performance. but there're certain cases where we need to police the long-runnings in protection of apiserver. added this to the non-blocking list.